arch/xtensa/src/esp32/Kconfig

```diff
@@ -5,6 +5,26 @@
 
 if ARCH_CHIP_ESP32
 
+choice ESP32_DEFAULT_CPU_FREQ_MHZ
+	prompt "CPU frequency"
+	default ESP32_DEFAULT_CPU_FREQ_240
+	help
+		CPU frequency to be set on application startup.
+
+	config ESP32_DEFAULT_CPU_FREQ_80
+		bool "80 MHz"
+	config ESP32_DEFAULT_CPU_FREQ_160
+		bool "160 MHz"
+	config ESP32_DEFAULT_CPU_FREQ_240
+		bool "240 MHz"
+endchoice # CPU frequency
+
+config ESP32_DEFAULT_CPU_FREQ_MHZ
```

Comment:
Why not have an int choice?

Reply:
Good suggestion David, and I think it was wrong, because we have a choice and a config with same name: ESP32_DEFAULT_CPU_FREQ_MHZ

Reply:
@davids5 we cannot get the selected value directly, more info: https://stackoverflow.com/questions/31325785/is-it-possible-to-convert-a-choice-to-an-int-in-kconfig